tests/conf/vhr10_ins_seg.yaml

```diff
@@ -9,6 +9,6 @@ data:
   init_args:
     batch_size: 1
     num_workers: 0
-    patch_size: 4
+    patch_size: 256
```

Comment:
This had to get fixed because the GeneralizedRCNNTransform was resizing behind the scenes. Now that that is no longer happening because it's a no-op we need to increase the patch size so the test images are resized.